briefcase-forms.rst

```diff
@@ -88,24 +88,24 @@ Pulling from Android 4.x and later device
 - Once it is copied onto your local hard drive, unzip the file of the 'odk' directory.
 - Open *ODK Briefcase*.
 - In the window which opens up select :guilabel:`pull` tab.
-- Select *Custom Path to ODK Directory* in the :guilabel:`pull data from` drop-down.
+- Select *Custom Path to ODK Directory* in the :guilabel:`pull data from` the drop-down.
```

Comment:
i don't understand this addition.

Reply:
Yes, I misinterpreted the text. I'll correct it!